cmd/openshift-install/create.go

```diff
@@ -264,10 +315,10 @@ func addRouterCAToClusterCA(ctx context.Context, config *rest.Config, directory
 	return nil
 }
 
-func waitForBootstrapComplete(ctx context.Context, config *rest.Config) (err error) {
+func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *clusterCreateError {
```

Comment:
If we are returning a pointer, then the receiver for the `unwrap` and `causeDetail` functions should be pointers, too.